docs/conf.py

```diff
@@ -11,11 +11,6 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-try:
```

Comment:
We don't want to keep this? I actually added this recently to pulp_rpm because the left-side navigation was pretty great.

Reply:
I am not sure I understand what this provides. This is not used by pulp-python and it's left menu seems to be the same as on pulp-rpm docs. What am I missing?

https://pulp-rpm.readthedocs.io/en/latest/index.html
https://pulp-python.readthedocs.io/en/latest/index.html

Reply:
AIUI, when rtd builds it, it applies the theme then so the rendered version there will look the same. What won't look the same is if you try to build it locally, it will get the sphinx default theme. These themes handle navigation differently so you can't develop the docs locally in a way that will look the same in production without this.

Reply:
Makes sense. I'll bring it back.